src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -126,6 +132,7 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
   def getUserStatus(userId: WorkbenchUserId, userDetailsOnly: Boolean = false, samRequestContext: SamRequestContext): Future[Option[UserStatus]] = {
     directoryDAO.loadUser(userId, samRequestContext).unsafeToFuture().flatMap {
       case Some(user) =>
+        logger.debug(s"Found user with id ${user.id}")
```

Comment:
Would prefer a little more context here.  I could imagine this message popping up all over the place.  
```suggestion
        logger.debug(s"getUserStatus - Found user with id ${user.id}")
```

Something like this.  Again, word it however you prefer.